src/ProjectTemplates/test/Templates.Tests/Templates.Tests.csproj

```diff
@@ -16,7 +16,7 @@
     <!-- Properties that affect test runs -->
     <!-- TestTemplateCreationFolder is the folder where the templates will be created. Will point out to $(OutputDir)$(TestTemplateCreationFolder) -->
     <TestTemplateCreationFolder>TestTemplates\</TestTemplateCreationFolder>
-    <TestPackageRestorePath>$([MSBuild]::EnsureTrailingSlash('$(RepoRoot)'))obj\template-restore\</TestPackageRestorePath>
+    <RestoreFolderName>Other</RestoreFolderName>
```

Comment:
@DamianEdwards I don't particularly like this name because it seems too generic. Is there anything these tests have in common aside from not being in the other categories / test projects❔

Reply:
Not really, they're just the "other" template tests that aren't Blazor or MVC, including the baseline tests, gRPC, empty web, etc.